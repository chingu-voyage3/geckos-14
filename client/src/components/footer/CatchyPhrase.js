import React from 'react';
import { Link } from 'react-router-dom';

const CatchyPhrase = () => {
   return (
        <div className="footer-catchyphrase">
          <p><strong>DashThings.io - An easy way to build <Link to="/Dashboards">Dashboards</Link> for your things.</strong></p>
        </div>
    );
};

export default CatchyPhrase;
