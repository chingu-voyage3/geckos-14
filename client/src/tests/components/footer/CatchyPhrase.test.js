import React from 'react';
import ReactDOM from 'react-dom';
import CatchyPhrase from '../../../components/footer/CatchyPhrase';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<CatchyPhrase />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
