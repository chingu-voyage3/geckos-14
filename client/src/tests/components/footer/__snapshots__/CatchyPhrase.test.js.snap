// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="footer-catchyphrase"
>
  <p>
    <strong>
      DashThings.io - An easy way to build 
      <Link
        replace={false}
        to="/dashboard"
      >
        Dashboards
      </Link>
       for your things.
    </strong>
  </p>
</div>
`;
